official/README.md

```diff
@@ -22,3 +21,11 @@ Below is a list of the models available.
 More models to come!
 
 If you would like to make any fixes or improvements to the models, please [submit a pull request](https://github.com/tensorflow/models/compare).
+
+---
+
+Run the models.
+
+The *Official Models* is made as a Python module. To run the models directly, you must change your work directory to the top-level ***/models*** folder (the parent directory of ***/official***), and set the Python path by issuing the following command: ``` export PYTHONPATH="$PYTHONPATH:$(pwd)" ```
```

Comment:
While it's true that this will work, I think it's a little atypical to request that people actually `cd` in first. I think it might be preferable to instruct people that they have to add the top-level models folder to their python path, and illustrate like so:

```
export PYTHONPATH="$PYTHONPATH:/path/to/models"
```

Also, nit: in Github markdown, you should be able to use one backtick ( ` ) instead of three ( ``` ) for in-line code snippets.

Reply:
Thanks for the comments, Karmel! 

For py3 test, I have submitted a CL to set up the python path, like what we did for py2. It should pass py3 tests once the CL is approved.

Reply:
@robieta Thank you for the detailed comments! Karmel also mentioned the same issue to me. Will update the branch as a single commit.